modules/integration_azure-load-balancer/conf/01-healthprobe.yaml

```
@@ -0,0 +1,21 @@
+---
+module: "Azure Load Balancer"
+name: backend unhealthy host ratio
+filtering: "filter('resource_type', 'Microsoft.Network/loadBalancers') and filter('primary_aggregation_type', 'true')"
+aggregation: ".mean(by=['BackendIPAddress', 'azure_resource_name', 'azure_resource_group_name', 'azure_region'])"
```

Otherwise, having a backend unhealthy one minute and healthy again will trigger the detector.

```suggestion
aggregation: ".max(by=['BackendIPAddress', 'azure_resource_name', 'azure_resource_group_name', 'azure_region'])"
```